docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,83 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |set certain contacts as favourites |access to their information easily
+
+|`* * *` |user |select multiple contacts to delete |clean up address book efficiently
+
+|`* * *` |user |save more than one contact number per contact in my address book |account for multiple phone numbers
+
+|`* * *` |organised user |group contacts manually or automatically |access the information I want easily
+
+|`* * *` |user |see my important events on a calendar view |overview of my schedule for the month
+
+|`* * *` |user |share my contact details with my friends |friends do not need to manually add it in
```

Comment:
(I want to ) exchange contacts with my friends | (so that I can) save the contacts without manually keying in the details.
This one I'm not sure how to rephrase but yah feel free to edit it. It's just a suggestion :)

Reply:
Done